svgo.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
           // customize options
           convertPathData: {
             // 3 decimals of precision in floating point numbers
-            floatPrecision: 3,
+            floatPrecision: 4,
```

Comment:
```suggestion
            floatPrecision: 3,
```

Please restore the floatPrecision to 3 in the svgo config before we merge this.